docker/data-science/notebooks/solr.py

```diff
@@ -349,36 +330,39 @@ def enable_ltr(self, collection):
         print(f"Adding LTR Doc Transformer for {collection.name} collection")
         response = requests.post(collection_config_url, json=del_ltr_transformer)
         response = requests.post(collection_config_url, json=add_transformer)
-        self.print_status(response.json())    
-
+        self.print_status(response.json())
+    
     def delete_feature_store(self, collection, name):
-        return requests.delete(f"{SOLR_URL}/{collection.name}/schema/feature-store/{name}")
+        return requests.delete(f"{SOLR_URL}/{collection.name}/schema/feature-store/{name}").json()
 
-    def create_feature_store(self, collection, features):
-        return requests.put(f"{SOLR_URL}/{collection.name}/schema/feature-store", json=features)
+    def upload_features(self, collection, features):
```

Comment:
Two things:
1. See previous comments about naming of model / feature upload / delete having "ltr" added to the function names.
2. These methods should probably not return the native Solr response since they're being used effectively as an engine-agnostic interface. i.e. maybe return a Boolean instead?